sdk/core/perf.yml

```
@@ -0,0 +1,32 @@
+parameters:
+- name: PackageVersions
+  displayName: PackageVersions (regex of package versions to run)
+  type: string
+  default: '1|source'
```

```suggestion
  default: 'source'
```

I believe C++ currently only supports testing the source code in the branch (not a published package).  If so, this parameter should only be `source` until published packages are supported.